templates/telemetry/update/update.4.parquetmr.txt

```diff
@@ -1,6 +1,17 @@
 message update {
   required binary id (UTF8);
   required binary clientId (UTF8);
+  required group application {
```

Comment:
Common sections should be updated here https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/dev/templates/include/telemetry/ and included.  If displayVersion is truly required you will need to create a new version of the application template.

Reply:
@trink what do you think about just including the one field that's needed here per [this comment](https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/88/files#r143238502) ? It seems preferable to repeating a whole bunch of redundant fields, or having to backfill to move things around.

Reply:
Won't this be a problem with Athena if you want to add another field later?

Reply:
Everything else from the `application` section is already present in the `environment`.